action.yml

```diff
@@ -1,36 +1,58 @@
 name: 'Dagger for GitHub'
-description: 'GitHub Action for Dagger, a programmable deployment system'
-author: 'dagger'
-branding:
-  color: 'green'
-  icon: 'package'
-
+description: 'Run dagger commands in Github Actions'
 inputs:
   version:
-    description: 'Dagger CUE version'
-    default: '0.2.232'
+    description: 'Dagger Version'
     required: false
-  args:
-    description: 'Arguments to pass to Dagger'
+    default: '0.9.3'
+  dagger-flags:
+    description: 'Dagger CLI Flags'
     required: false
-    deprecationMessage: 'Use cmds input instead'
-  cmds:
-    description: 'List of Dagger commands'
+    default: '--progress plain'
+  verb:
+    description: 'CLI verb (call, download, up, functions, shell, query)'
     required: false
+    default: 'call'
   workdir:
-    description: 'Working directory (below repository root)'
+    description: 'The working directory in which to run the Dagger CLI'
+    required: false
     default: '.'
+  cloud-token:
+    description: 'Dagger Cloud Token'
     required: false
-  install-only:
-    description: 'Just install Dagger'
-    default: 'false'
+    default: ''
+  module:
+    description: 'Dagger module to call. Local or Git'
     required: false
-  cleanup:
-    description: 'Cleanup Dagger home folder at the end of a job'
-    default: 'true'
+    default: ''
+  args:
+    description: 'Arguments to pass to CLI'
     required: false
-
+    default: ''
 runs:
-  using: 'node16'
-  main: 'dist/index.js'
-  post: 'dist/index.js'
+  using: "composite"
+  steps:
+    - run: |
+        cd /usr/local && { \
+        curl -sL https://dl.dagger.io/dagger/install.sh 2>/dev/null | \
+        DAGGER_VERSION=${{ inputs.version }} sh 2>/dev/null; }
+      shell: bash
+
+    - run: if ! [ -d ./.git ]; then git clone -b ${GITHUB_REF_NAME} --no-checkout https://github.com/${GITHUB_REPOSITORY} .; fi
```

Comment:
Is this so we can get the git metadata for cloud?

Makes sense for now if so, but one of the cool benefits is that the git checkout should be optional now. With some updates to how we get the git metadata (i.e. optionally parse from env rather than the local `.git`) we could achieve that I think. Obviously not a blocker here, just checking that's the reason we included this.

Reply:
I'll try it without this and see if it's really needed for Dagger Cloud.

Reply:
If I remove the git stuff
![image](https://github.com/dagger/dagger-for-github/assets/3187222/f2077ad7-361c-4b06-bd00-5d3c570f8814)

Reply:
Here's a run that still has the git script
![image](https://github.com/dagger/dagger-for-github/assets/3187222/229d6eab-c096-42ad-a633-2e29f51645f8)

Reply:
Yeah, I don't see any reason to keep it. It's outta here!